test/nbrowser/DocTypeConversion.ts

```diff
@@ -0,0 +1,187 @@
+import { UserAPI } from "app/common/UserAPI";
+import { assert, driver, until } from "mocha-webdriver";
+import * as gu from "test/nbrowser/gristUtils";
+import { setupTestSuite } from "test/nbrowser/testUtils";
+import { Button, button, element, label, option} from "test/nbrowser/elementUtils";
+
+type TypeLabels = "Regular" | "Template" | "Tutorial";
+
+
+describe("Document Type Conversion", function () {
```

Comment:
```suggestion
describe("DocTypeConversion", function () {
```
Names of the primary `describe` should match name of the file. This is a convention we have, this way it is easier to use GREP_TESTS which finds tests by this name, not the file name.